client/src/App.js

```diff
@@ -15,7 +16,8 @@ function App() {
         <Route path="/" exact component={LandingPage} />
         <Route path="/login" component={Login} />
         <Route path="/signup" component={SignUp} />
-        <Route path="/list" component={List} />
+					{/* <Route path="/board" component={Board} /> */}
```

Comment:
Can you merge dev into this and then uncomment these lines?

Reply:
I made a few changes after your comment was made. Would you still like me to merge my branch into dev?